src/Renci.SshNet/Sftp/SftpSession.cs

```diff
@@ -472,6 +541,33 @@ public void RequestClose(byte[] handle)
             }
         }
 
+#if FEATURE_TAP
+        public async Task CloseAsync(byte[] handle, CancellationToken cancellationToken)
```

Comment:
Please rename to **RequestCloseAsync** and move next to **RequestClose** method.